src/components/ChatLog.jsx

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import messages from '../data/messages';
+
+const localUser = messages.length > 0 ? messages[0].sender : null;
```

Comment:
This logic feels like it could be placed inside of `App.jsx` to me since you are already importing the `messages` there and just pass it down to `ChatLog.jsx`. Imagine if this was an API call.